webpages/settings/index.html

```diff
@@ -174,6 +174,17 @@ <h1>{{ msg("settings") }}</h1>
                     :placeholder="setting.default"
                     maxlength="100"
                   />
+                  <input
+                    type="text"
+                    class="setting-input key"
+                    v-model="addonSettings[addon._addonId][setting.id]"
+                    @input="updateSettings(addon)"
+                    :disabled="!addon._enabled"
+                    v-if="setting.type === 'key'"
+                    :placeholder="setting.default"
+                    maxlength="100"
+                    :id="setting.id"
```

Comment:
This could break if 2 addons have the same setting ID

Reply:
Who would make an addon that has the same as another?

Reply:
This is for the *setting* ID, not the addon ID. Setting IDs must be unique for each addon, but might not be unique for all addons.